build.rs

```diff
@@ -0,0 +1,18 @@
+use std::env;
+use std::fs;
+use std::path::Path;
+
+fn main() {
```

Comment:
Wouldn't it be neater to use `?` here everywhere instead of unwrap and have `main()` just return an `anyhow::Result`?